test/unit/api/CMakeLists.txt

```diff
@@ -21,3 +21,8 @@ cvc4_add_unit_test_white(solver_white api)
 cvc4_add_unit_test_black(sort_black api)
 cvc4_add_unit_test_black(term_black api)
 cvc4_add_unit_test_white(term_white api)
+
+# add Python bindings tests if building with Python bindings
+if (BUILD_BINDINGS_PYTHON)
+  add_subdirectory(python)
+endif()
```

Comment:
I'm not sure if we should make a python subdirectory in `test/unit/api/python` or do something like `test/python/unit/api`. We already have a `test/api/python` directory. I'd suggest we use `test/<lang>/{unit,api,...}` in order to avoid all the `python` subdirectories. This also has the advantage that we need to check for `pytest` once in `test/python/CMakeLists.txt`.

Reply:
I think the idea was to move the tests from `test/api` but regardless you still make a good point. Eventually there might be multiple directories of Python tests and it makes sense to gather them.

Reply:
yes, I was planning to move the content of `test/api/python` to `test/unit/api/python`. Will we have any other python tests except for unit tests of the python api?

Reply:
If the only tests will be in `test/unit/api` I'm fine with having a `test/unit/api/{python,java,...}` directory structure.

Reply:
@makaimann @aniemetz do we expect other python tests?

Reply:
I’m not sure, but my inclination is to make it flexible enough to add (non-unit) tests in the future. Seems safer to allow the possibility than assume it won’t ever be added imo.

Reply:
OK that makes sense. I have reorganized. Hopefully the way I handled `cmake` is not too silly (one liner `CMakeLists.txt` files that just add the only subdir). Please let me know if there is a better way.